src/restic_fmt_plug.c

```diff
@@ -0,0 +1,334 @@
+/*
+ * This software is Copyright (c) 2020, Jürgen Hötzel <juergen at hoetzel.info>
+ * and it is hereby released to the general public under the following terms:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted.
+ */
+
+#if FMT_EXTERNS_H
+extern struct fmt_main fmt_restic;
+#elif FMT_REGISTERS_H
+john_register_one(&fmt_restic);
+#endif
+
+#include <string.h>
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+#include "aes.h"
+#include "arch.h"
+#include "base64_convert.h"
+#include "common.h"
+#include "formats.h"
+#include "jumbo.h"
+#include "misc.h"
+#include "options.h"
+#include "params.h"
```

Comment:
What does this file use from these? -

```c
#include "jumbo.h"
#include "options.h"
#include "params.h"
```

(I see it uses `strnzcpy()` from `misc.h`.)

Reply:
Sorry this was just leftover from `ethereum_fmt_plug.c` (I used as boilerplate).

I have removed the unnecessary includes.

Reply:
We should probably look into cleaning up `ethereum_fmt_plug.c` as well, although a few things you had inherited from there and now dropped from here were in fact needed there (I think SIMD alignment and going over groups of `MIN_KEYS_PER_CRYPT` keys, as they were for older Ethereum "hashes" processed by a SIMD implementation of PBKDF2).